.github/workflows/test-erlang.yml

```
@@ -0,0 +1,100 @@
+name: Test Erlang
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'epic/**'
+  pull_request:
+    branches: [ '**' ]
+
+env:
+  JAVA_VERSION: 11
+  OTP_VERSION: "24.2.0"
+  REBAR_VERSION: "3.18"
+  CODEGEN_VERSION: "2.4.25"
+  CODEGEN_NAME: "swagger-codegen-cli.jar"
+  ARTIFACT_NAME: "vality-erlang-codegen-swagger-codegen-1.0.0.jar"
+  SPEC_URL: "https://petstore.swagger.io/v2/swagger.json"
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+          distribution: 'adopt'
+
+      - name: Get Codegen
+        run: |
+          mkdir codegen-cli
+          echo "DOWNLOADING ${{ env.CODEGEN_VERSION }}"
+          wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${{ env.CODEGEN_VERSION }}/swagger-codegen-cli-${{ env.CODEGEN_VERSION }}.jar -O codegen-cli/${{ env.CODEGEN_NAME }}
+
+      - name: Build package
+        run: |
+          mvn package
```

```suggestion
          mvn --batch-mode package
```

* https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven#using-the-maven-starter-workflow
* https://stackoverflow.com/a/21638827